### PR TITLE
Fixed math error and improved language

### PR DESCRIPTION
**Pull request by geraldwenzel**
Swapped **precision** for **accuracy** because within the context of numerical calculation accuracy refers to how close the calculated answer is to the correct value, while precision refers to the number of significant figures used in the result. 

This statement has a high level of precision but low accuracy:
$$pi = 5.464584634564$$

This statement has lower precision but higher accuracy:
$$pi = 3.14$$

The purpose of multiplying before dividing is to calculate a result with higher accuracy.

---
`ethPrice * ethAmount results in 2000e18` is false because:
$$1e18 * 2000e18 = 2000e36$$

### CODE REVIEW

**Comment by EngrPips** [comment]
Hello @geraldwenzel, Thanks for this contribution. It will be addressed appropriately as soon as possible.

File: courses/solidity/3-fund-me/11-more-solidity-math/+page.md [hunk 1]
```diff
@@ -29,13 +29,13 @@ function getConversionRate(uint256 ethAmount) internal view returns (uint256) {
 > The line `uint256 ethAmountInUsd = (ethPrice * ethAmount)` results in a value with a precision of 1e18 \* 1e18 = 1e36. To bring the precision of `ethAmountInUsd` back to 1e18, we need to divide the result by 1e18.
 
 > 🔥 **CAUTION**:br
-> Always multiply before dividing to maintain precision and avoid truncation errors. For instance, in floating-point arithmetic, `(5/3) * 2` equals approximately 3.33. In Solidity, `(5/3)` equals 1, which when multiplied by 2 yields 2. If you multiply first `(5*2)` and then divide by 3, you achieve better precision.
+> Always multiply before dividing to maintain precision and avoid truncation errors. For instance, in floating-point arithmetic, `(5/3) * 2` equals approximately 3.33. In Solidity, `(5/3)` equals 1, which when multiplied by 2 yields 2. If you multiply first `(5*2)` and then divide by 3, you achieve better accuracy.
```
Comment (Equious) [inline courses/solidity/3-fund-me/11-more-solidity-math/+page.md:32]: While perhaps technically true, I think 'precision' is used colloquial in the space for these circumstances.

**Review by Equious — CHANGES_REQUESTED** [review]
Small adjustment!

File: courses/solidity/3-fund-me/11-more-solidity-math/+page.md (hunk 1 above)
Comment (geraldwenzel) [reply courses/solidity/3-fund-me/11-more-solidity-math/+page.md]: Submitted change per feedback.

**Review by Equious — APPROVED** [review]
Thanks!

**Pull request merged**